### ISSUE TITLE
MDDatable `row_num` should be set to inifinity when `use_pagnition` is set to False

### ISSUE THREAD

**Issue by kengoon** [kivymd/KivyMD#1216]
when `use_pagnition = False`, datatable should not be limited by `row_num` rather it should be set to infinity

**Comment by epitavy** [comment]
I fully agree, I was stuck on my MDDataTable displaying only five rows with `use_pagination` set to False.

Please edit the title and change True to False.
